configuration.yaml

```diff
@@ -89,6 +90,10 @@ immunomodulator_groups:
   - "Immune_Checkpoint"
   - "Function"
   
+immunotarget_groups:
+  - "Therapy Type"
+  - "Pathway"
+
```

Comment:
@andrewelamb — any potential problems introduced here with spaces instead of underscores?

Reply:
Not from a yaml standpoint, it depends what the module is doing with these.